ui/src/features/annotator/mask-annotations.tsx

```diff
@@ -3,40 +3,22 @@
 
 import { ReactNode, useId } from 'react';
 
-import { AnnotationShape } from './annotation-shape';
-import { Annotation } from './types';
-
 type MaskAnnotationsProps = {
-    annotations: Array<Annotation>;
     children: ReactNode;
     width: number;
     height: number;
     isEnabled: boolean;
 };
 
-export const MaskAnnotations = ({ annotations, children, width, height, isEnabled }: MaskAnnotationsProps) => {
+export const MaskAnnotations = ({ children, width, height, isEnabled }: MaskAnnotationsProps) => {
     const id = useId();
+
     const maskOpacity = isEnabled ? 0.8 : 0.0;
 
     return (
         <>
             <mask id={`mask-${id}`}>
                 <rect x='0' y='0' width={width} height={height} style={{ fill: 'white', fillOpacity: 1.0 }} />
-                {annotations.map((annotation, idx) => (
```

Comment:
Can you clarify this change?

Reply:
The annotations were being rendered here and below on this file. I dont know why we needed this duplication? The visual effect is the same. Maybe i missed something? I made this change on a single commit. Perhaps @camiloHimura  will know more?